build/NuGetAdditionalFiles/RunCsc

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
I can imagine people in about 5 seconds taking a dependency on the existence of this script in the package. "Oh, I downloaded the package, I guess I run it via running this script..."